packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -108,6 +110,20 @@ function GridItem( { categoryId, item, ...props } ) {
 	};
 	const deleteItem = () =>
 		isTemplatePart ? removeTemplate( item ) : deletePattern();
+	const exportAsJSON = () => {
+		const json = {
+			__file: item.type,
+			title: item.title,
+			content: item.reusableBlock.content.raw,
+			syncStatus: item.reusableBlock.wp_pattern_sync_status,
```

Comment:
```suggestion
			content: item.patternBlock.content.raw,
			syncStatus: item.patternBlock.wp_pattern_sync_status,
```

After the rebase the `reusableBlock` property doesn't exist so this throws an error. The suggested tweak fixes it for me.